back-end/API/app/controllers/jobOffer_controller.py

```
@@ -1,159 +1,100 @@
+    dto = JobOfferApproveDTO.model_validate(request.get_json())
+    dto.id = id
+    job_offer = job_offer_service.approve(dto)
+    return job_offer.model_dump(), 200
 
 @job_offer_blueprint.route('/archive/<int:id>', methods=['POST'])
 @token_required
-def archiveJobOffer(current_user, id):
-    jobOffer_service.archiveJobOffer(id)
-    return ('', 204)
-    
+def archive_job_offer(current_user, id):
```

Ruff (0.15.13)

[warning] Unused function argument: `current_user`

(ARG001)